en/integrations/plugin-packs/procedures/network-switchs-symbol-wing-snmp.md

```diff
@@ -0,0 +1,178 @@
+| Net-Symbol-Wing-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
+
+### Collected metrics & status
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Systems-->
+
+* Global
+
+| Metric name         | Description             | Unit  |
+|:--------------------|:------------------------|:------|
+| devices.total.count | Number of total devices | count |
+
+* Per *cpu*
+
+| Metric name                    | Description       | Unit |
+|:-------------------------------|:------------------|:-----|
+| cpu.utilization.1m.percentage  | Cpu usage for 1m  | %    |
+| cpu.utilization.5m.percentage  | Cpu usage for 5m  | %    |
+| cpu.utilization.15m.percentage | Cpu usage for 15m | %    |
+
+* Per *memory*
+
+| Metric name                    | Description               | Unit |
+|:-------------------------------|:--------------------------|:-----|
+| device.memory.usage.bytes      | Used memory               | B    |
+| device.memory.free.bytes       | Free memory               | B    |
+| device.memory.usage.percentage | Percentage of used memory | %    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+No specific prerequisites
+
+### Network flow
+
+The target server must be reachable from the Centreon Poller on the UDP/161 SNMP
+port.
+
+## Setup
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Symbol WiNG** ressources:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. On the Centreon Web interface, install the **Symbol WiNG switch SNMP** Centreon Plugin Pack on the **Configuration > Plugin Packs** page.
+
+<!--Offline License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor **Symbol WiNG** ressources:
+
+```bash
+yum install centreon-plugin-Network-Symbol-Wing-Snmp
+```
+
+2. Install the **Symbol WiNG switch SNMP** Centreon Plugin Pack RPM on the Centreon Central server:
+
+ ```bash
+yum install centreon-pack-network-switchs-symbol-wing-snmp
+```
+
+3. On the Centreon Web interface, install the **Symbol WiNG switch SNMP** Centreon Plugin Pack on the **Configuration > Plugin Packs** page.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**.
+* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your **Symbol WiNG** server settings.
+* Select the **Net-Symbol-Wing-SNMP-custom** template to apply to the Host
+
+If you are using SNMP Version 3, use the SNMPEXTRAOPTIONS Macro to configure
+your own SNMPv3 credentials combo.
+
+| Mandatory | Name             | Description                                              |
+|:----------|:-----------------|:---------------------------------------------------------|
+|           | SNMPEXTRAOPTIONS | (Default: 'Configure your own SNMPv3 credentials combo') |
+
```

Comment:
Oops my bad, it will be fixed in the script.
```suggestion
| Mandatory | Name             | Description                                 |
|:----------|:-----------------|:--------------------------------------------|
|           | SNMPEXTRAOPTIONS | Configure your own SNMPv3 credentials combo |

```